signalfx-smart-gateway/README.md

```diff
@@ -46,6 +46,8 @@ The downloaded checksum assumes that the Smart Gateway binary is in the current
 
 #### Changelog
 
+Note: To use the following download links, you must be viewing this content from within SignalFx. Instructions for manually downloading a specific version of the Smart Gateway are shown [above](#installation).
```

Comment:
<from within SignalFx> ==> <from within the SignalFx UI>

I've seen some examples of "UI" in the doc site, which I think helps to make this statement clearer.